wordpress-mobile/WordPress-Android
# Stats revamp v2:  Remove feature config

This PR remove **StatsRevampV2 Feature Config**, **Stats New Feature Intro**  and updates all the places that was in use including analytics and prefs.

Fixes #16988 

To test:

- Build and run
- Launch **Jetpack app**
- Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
- Select Debug Settings
- Ensure **StatsRevampV2FeatureConfig** is not present
- Ensure no Stats Feature Intro is shown
- Navigate to Stats, and ensure the new cards are shown or a pop-up is displayed about update and new cards shown
- Also, launch **WordPress app** and ensure it is still showing old stats cards on Stats screen.



## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
updated unit tests

PR submission checklist:
